### ISSUE TITLE
Unable to install yt-dlp (Windows 10 Installer - V2023.12.0)

### ISSUE THREAD

**Issue by CarrotManMatt** [NickvisionApps/Parabolic#719]
Following on from the install issues described in #687, the Windows 10 install fails with [V2023.12.0](https://github.com/NickvisionApps/Parabolic/releases/tag/2023.12.0) due to: `Unable to install yt-dlp.`

![image](https://github.com/NickvisionApps/Parabolic/assets/108596565/b647cc80-b690-492c-8268-ece14612af1b)

**Comment by nlogozzo** [comment]
Really?! Wow.

Maybe it's a pip issue and not a psutil one. Will look into it...

**Comment by nlogozzo** [comment]
@CarrotManMatt This may sound like a weird question, but did you change the default install location for programs? Like is it not `C:\Program Files`?

**Comment by CarrotManMatt** [comment]
I have no `Python311` folder in `C:\Program Files`, nor in `C:\Program Files (x86)`

**Comment by nlogozzo** [comment]
Ah ok so then it's python itself not installing correctly...

Have you ever installed python before or have another version installed? Through Microsoft store or wherever?

**Comment by CarrotManMatt** [comment]
Yeah I used to have 3.11 installed but then removed that & updated to 3.12. I know that me manually changing my Python version is probably causing these issues, but would it not be best for Parabolic to be independent from *any* Python versions the user may have installed? (Just my own opinion/suggestion)

**Comment by nlogozzo** [comment]
Well yt-dlp doesn't (or didn't at the time when we made the decision) work with 3.12, hence why we used 3.11 

Plus installing a fresh version for aparabolic ensures dependencies and everything get setup correctly by us. 

Python is supposed to allow multiple versions anyway should it should be a problem, but here 3.11 isn't installing.

Maybe I will change the install location to a more Parabolic centric location...

Thanks for the info, I'll try to get a new installer to you tonight.
